cdci_data_analysis/flask_app/app.py

```diff
@@ -604,6 +604,11 @@ def validate_schema(response):
             'error': repr(e),
             'invalid_response': response.json
         }), 500
+    # TODO improve this
+    response.headers.add('Access-Control-Allow-Origin', '*')
+    response.headers.add('Access-Control-Allow-Headers', 'Content-Type,Authorization')
+    response.headers.add('Access-Control-Allow-Methods', 'GET,POST,PUT,DELETE,OPTIONS')
```

Comment:
I would be extra-careful when setting CORS headers for every response of the app. Better to set for the one that really requires it.

Reply:
this is, obviously, a first version, that I right now use only for tests, of course it has to better handled